src/Slices/ServiceInventory/UI/Components/RowActionsMenu/RowActions.tsx

```diff
@@ -166,11 +168,11 @@ export const RowActions: React.FunctionComponent<InstanceActionsProps> = ({
               service: instance.service_entity,
               instance: instance.id,
             })}
-            isDisabled={editDisabled}
+            isDisabled={editDisabled || composerDisabled}
```

Comment:
Do we want to disable it or hide it 🤔 I think hiding it would be more meaningful. There's no case where the button might go from disabled to enabled in the same view, unlike the edit button for example that can become enabled because of a state change.